### PR TITLE
Add attributes s3_key_prefix & organization_trail to aws_cloudtrail_trail resource

### PR DESCRIPTION
**Pull request by rezen**
### Description
Added attributes `s3_key_prefix` & `organization_trail` to aws_cloudtrail_trail resource

### Check List
Please fill box or appropriate ([x]) or mark N/A.
- [x] New functionality includes integration tests/controls
- N/A New Terraform resources
- [x] Documentation provided or updated for resources 
- [x] All Integration Tests pass
- [x] All Unit Tests pass
- [x] `rake lint` passes
- [x] All commits have been signed-off for the Developer Certificate of Origin. See <https://github.com/chef/chef/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco>

### CODE REVIEW

File: libraries/aws_cloudtrail_trail.rb [hunk 1]
```diff
@@ -10,11 +10,13 @@ class AwsCloudTrailTrail < AwsResourceBase
...
 
   attr_reader :cloud_watch_logs_log_group_arn, :cloud_watch_logs_role_arn, :home_region, :trail_name,
-              :kms_key_id, :s3_bucket_name, :trail_arn, :is_multi_region_trail, :log_file_validation_enabled
+              :kms_key_id, :s3_bucket_name, :s3_key_prefix, :trail_arn, :is_multi_region_trail,
+              :log_file_validation_enabled, :is_organization_trail
 
   alias multi_region_trail? is_multi_region_trail
   alias log_file_validation_enabled? log_file_validation_enabled
   alias has_log_file_validation_enabled? log_file_validation_enabled
+  alias organization_trail? is_organization_trail
```
Comment (soumyo13) [inline libraries/aws_cloudtrail_trail.rb:19]: Can we remove this? We dont require the alias. is_organization_trail looks good.

**Comment by rezen** [comment]
@soumyo13 I did the sign off, I think it's unhappy because I didn't include my email in `Signed-off-by: Andres Hermosilla <>`

**Comment by soumyo13** [comment]
@rezen Please add the email in the sign off. I think that will work and DCO will pass.

**Comment by rezen** [comment]
@soumyo13 Fixed the DCO 👍

File: libraries/aws_cloudtrail_trail.rb (hunk 1 above)
Comment (soumyo13) [reply libraries/aws_cloudtrail_trail.rb:19]: I tried to run with the control with organization_trail, it failed. It is not taking the alias.

Code that I have used is 

control "test1" do
  describe aws_cloudtrail_trail('testbucketname') do
    its("is_organization_trail") { should eq false }
    its("organization_trail") { should eq false }
    its("organization_trail") { should eq "" }
  end
end

output

×  test1: CloudTrail testbucketname (2 failed)
     ✔  CloudTrail testbucketname is_organization_trail is expected to eq false
     ×  CloudTrail testbucketname organization_trail is expected to eq false

     expected: false
          got: #<#<Class:0x000000010fc0d860>::NullResponse:0x000000010fa83198>

     (compared using ==)
     ×  CloudTrail testbucketname organization_trail is expected to eq false

     expected: false
          got: #<#<Class:0x000000010fc0d860>::NullResponse:0x000000010fa83198>

     (compared using ==)

Comment (soumyo13) [reply libraries/aws_cloudtrail_trail.rb:19]: @rezen Please check this.

Comment (clintoncwolfe) [reply libraries/aws_cloudtrail_trail.rb:19]: Try it { should be_organizational_trail } . It makes good English.

Comment (soumyo13) [reply libraries/aws_cloudtrail_trail.rb:19]: it { should be_organizational_trail } 

This is working fine. So we can merge this PR.

**Review by soumyo13 — APPROVED** [review]
LGTM

**Pull request merged**